docs/source/markdown/podman-network-create.1.md

```diff
@@ -7,8 +7,10 @@ podman\-network-create - Create a Podman CNI network
 **podman network create**  [*options*] name
 
 ## DESCRIPTION
-Create a CNI-network configuration for use with Podman. At the time of this writing, the only network
-type that can be created is a *bridge* network.
+Create a CNI-network configuration for use with Podman. By default, Podman creates a bridge connection. A
+*macvlan* connection can be created with the *macvlan* option. In the case of *macvlan* connections, the
+CNI *dhcp* plugin needs to be activated or the container image must have a DHCP client to interact
```

Comment:
How do we activate the DHCP plugin?

Reply:
that is out of scope for podman ... that said, i think our fedora cni package should ship the systemd service file